### DISCUSSION TITLE
Add accountState to return value of useWeb3Modal to get connected wallet address

### DISCUSSION THREAD

**Discussion by nob3110** [WalletConnect/walletconnect-monorepo#2442] [Developer Support] [answered]
Relating to: https://github.com/orgs/WalletConnect/discussions/2357

My ReactNative app wants to display the address of the user's wallet after establishing a connection.

Could you please add one line as follows?

https://github.com/WalletConnect/web3modal-react-native/blob/c2ae11cc135140734bc07d2a2e18c6b505c4c08e/src/hooks/useWeb3Modal.ts#L18

````
  return {
    isOpen: modalState.open,.
    open: ModalCtrl.open, ModalCtrl.open, ModalCtrl.close, ModalCtrl.close
    close: ModalCtrl.close, provider: clientState.initialized ?
    provider: clientState.initialized ? ClientCtrl.provider() : undefined,.
    isConnected: accountState.isConnected, // here
    accountState, // here
    clientState // someone else might use this?
  }; }
````

This is how my app got wallet address.
but, I don't understand all of the web3modal-react-native code, so I don't know if this is the best method.

**Comment by ignaciosantise** [comment, marked as answer]
@nob3110  `@web3modal/react-native` alpha.5 is now available, and it has the `address` prop in `useWeb3Modal` 💪

You can get the wallet address doing:
```tsx
const { address } = useWeb3Modal();
```

**Reply by nob3110** [reply]
@ignaciosantise  Thanks so much for your quick update!
